articles/protocols/oauth2/index.md

```diff
@@ -1,53 +1,53 @@
+> Many luxury cars today come with a valet key. It is a special key you give the parking attendant and unlike your regular key, will not allow the car to drive more than a mile or two. Some valet keys will not open the trunk, while others will block access to your onboard cell phone address book. Regardless of what restrictions the valet key imposes, the idea is very clever. You give someone limited access to your car with a special key, while using your regular key to unlock everything.
 
-- **Resource Owner**: the entity that can grant access to a protected resource. Typically this is the end-user.
+To get access to the protected resources OAuth 2.0 uses **access tokens**. An access token is a string in [JSON Web Token (JWT)](/jwt) format, that contains the list of authorized permissions.
 
-- **Resource Server**: the server hosting the protected resources. This is the API you want to access.
+::: panel-info Access Token Format
+By default, Auth0 generates access tokens in [JSON Web Token (JWT)](/jwt) format, an industry standard. JWTs contain three parts: a header, a payload, and a signature:
+ - The header contains metadata about the type of token and the cryptographic algorithms used to secure its contents.
+ - The payload contains a set of claims, which are statements about the permissions that should be allowed, and other information like the intended audience and the expiration time.
+ - The signature is used to validate that the token is trustworthy and has not been tampered with.
 
-- **Client**: an application requesting access to a protected resource on behalf of the Resource Owner.
+ For more information refer to [JSON Web Tokens (JWT) in Auth0](/jwt).
+:::
 
-- **Authorization Server**: the server that authenticates the Resource Owner, and issues access tokens after getting proper authorization. In this case, Auth0.
+The permissions contained in the access token, in OAuth 2.0 terms are known as **scopes**. When a client authenticates with Auth0, it specifies the scopes it wants. If those scopes are authorized by the user, then the access token will contain a list of authorized scopes.
 
+For example, a Contacts API may accept four different levels of authorization: reading contacts (scope `read:contacts`), creating contacts (scope `create:contacts`) and deleting contacts (scope `delete:contacts`). When a client asks the API to create a new contact, then the access token should contain the `create:contacts` scope. In a similar fashion, in order to delete existing contacts, the access token should contain the `delete:contacts` scope.
 
-## Access Token
+::: panel-info Scope Request Parameter
+The reason why we refer to the permissions as scopes is because these permissions are sent to the relevant endpoints using the `scope` request parameter. The value of this parameter is expressed as a list of space-delimited, case-sensitive, predefined strings.
```

Comment:
Not sure about this wording,

> these permissions are sent to the relevant endpoints using the `scope` request parameter.

What are the relevant endpoints? Or the request parameter? Maybe it's too soon to go technical.